.github/workflows/ci.yaml

```diff
@@ -0,0 +1,44 @@
+name: CI
+on:
```

Comment:
_(arbitrary comment location to start a thread)_

Should we delete the Makefile? Originally it was helpful because we tested against a forked network. Now it does nothing except alias the forge commands so I'm thinking we should remove it